.circleci/config.yml

```diff
@@ -1,9 +1,13 @@
 version: 2.1
 
+parameters:
+  releases-slack-channel:
+    type: string
+    default:  probation_in_court_dev
+
 orbs:
   owasp: entur/owasp@0.0.10
   hmpps: ministryofjustice/hmpps@10
-  slack: circleci/slack@4.1.1
```

Comment:
only used to notify on failed validation steps

This is only likely to happen when building branches for PRs.
When someone is working on a PR, they are likely to be checking whether the build passed so that they can ask for a review and get it merged
This alert seems unnecessary